lib/matplotlib/figure.py

```diff
@@ -1412,6 +1418,53 @@ def align_ylabels(self, axs=None):
                         # grouper for groups of ylabels to align
                         self._align_label_groups['y'].join(ax, axc)
 
+    def align_titles(self, axs=None):
+        """
+        Align the titles of subplots in the same subplot column if title
```

Comment:
```suggestion
        Align the titles of subplots in the same subplot row if title
```
Copy/paste error I think, as this function works on rows.  `align_xlabels` has the same problem so for bonus points you could fix that docstring too.